### PR TITLE
CTCP-5222: Departure Transport means mini CYA 

### PR DESCRIPTION
**Pull request by joecefnhedog**
image

### CODE REVIEW

File: conf/messages [hunk 1]
```diff
@@ -25,6 +25,8 @@ caption = Transport details
 
 checkYourAnswers.heading = Check your answers
 checkYourAnswers.title = Check your answers - Transport details
+miniCheckYourAnswers.heading = Check your answers
+miniCheckYourAnswers.title = Check your answers - Transport details
```
Comment (jackheslop96) [inline conf/messages:29]: Content doc suggests this should be `Check your answers - Departure means of transport`

Comment (jackheslop96) [reply conf/messages:29]: And for the sake of consistency can these keys be something like `transportMeans.departure.checkYourAnswers` ? `miniCheckYourAnswers` is a bit vague

File: conf/messages [hunk 2]
```diff
@@ -25,6 +25,8 @@ caption = Transport details
 
 checkYourAnswers.heading = Check your answers
 checkYourAnswers.title = Check your answers - Transport details
+transportMeans.departure.heading = Check your answers
```
Comment (kenneth-rayner) [inline conf/messages:28]: Should the caption/subheading not be 'Transport details - Departure means of transport'

Comment (kenneth-rayner) [inline conf/messages:28]: The 
Do you want to add the type of identification?
should be
Do you want to add the type of identification for the departure means of transport?

File: conf/messages [hunk 3]
```diff
@@ -25,6 +25,8 @@ caption = Transport details
...
+transportMeans.departure.title = Check your answers - Departure means of transport
```
Comment (kenneth-rayner) [inline conf/messages:29]: You haven't added the hidden text for the change links

File: test/viewModels/DepartureTransportAnswersViewModelProviderSpec.scala [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+      val viewModelProvider = new DepartureTransportAnswersViewModelProvider()
+      val result            = viewModelProvider.apply(userAnswers, index)
+
+      val section = result.sections.head
+      section.sectionTitle mustBe None
+      section.rows.size mustBe 5
+      section.addAnotherLink must not be defined
+    }
+
```
Comment (kenneth-rayner) [inline test/viewModels/DepartureTransportAnswersViewModelProviderSpec.scala:65]: Would we want 7 lines including addIdentitifcationYesNo page and addIdentificationNumberYesNo page?

File: conf/messages (hunk 3 above)
Comment (kenneth-rayner) [inline conf/messages:29]: should we not have the key:
transportMeans.departure.checkYourAnswers.title

File: conf/messages (hunk 2 above)
Comment (kenneth-rayner) [reply conf/messages:28]: should the key not be 
transportMeans.departure.checkYourAnswers.heading

**Review by kenneth-rayner — DISMISSED** [review]

**Review by kenneth-rayner — APPROVED** [review]

**Pull request merged**